web/packages/teleterm/src/services/tshd/types.ts

```diff
@@ -318,3 +322,9 @@ export type Label = apiLabel.Label.AsObject;
 
 // Replaces object property with a new type
 type Modify<T, R> = Omit<T, keyof R> & R;
+
+export type UpdateHeadlessAuthenticationStateParams = {
+  rootClusterURI: uri.RootClusterUri;
+  headlessAuthenticationID: string;
```

Comment:
Could you change them to:
`rootClusterUri` and `headlessAuthenticationId`? That's a convention in `web`.